packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2139,6 +2139,7 @@ export default mixins(
 	position: relative;
 	width: 100%;
 	height: 100%;
+	transform-origin: 0 0;
```

Comment:
One side effect of this styling change is that it zooms from upper left instead of center (which I personally prefer). I couldn't find an implementation solution to #getMousePositionWithinNodeView w/o this change. Related SO answer: https://stackoverflow.com/a/15521750.

Reply:
Just tested. Also seems OK to me.